lib/util/got/index.ts

```diff
@@ -1,17 +1,16 @@
+import got from 'got';
 import auth from './auth';
 import cacheGet from './cache-get';
 import hostRules from './host-rules';
-import renovateAgent from './renovate-agent';
 import { mergeInstances } from './util';
 
 export * from './common';
 
 /*
  * This is the default got instance for Renovate.
- *  - Set the user agent to be Renovate
  *  - Cache all GET requests for the lifetime of the repo
  *
  */
-export const api = mergeInstances(cacheGet, renovateAgent, hostRules, auth);
+export const api = mergeInstances(got, cacheGet, hostRules, auth);
```

Comment:
really strange 🤔

Reply:
All the other instances don't provide options, so you need to pass `got` as the first instance because it does have default options already.

Reply:
Thanks for your quick answering. That makes sense. We kind of "got lucky" and didn't realise that previously due to the user agent extension.